crates/engine/tree/src/tree/mod.rs

```diff
@@ -1794,11 +1809,6 @@ where
                 return Ok(TreeOutcome::new(outcome))
             }
 
-            if let Some(attr) = attrs {
-                let updated = self.process_payload_attributes(attr, &tip, state);
-                return Ok(TreeOutcome::new(updated))
-            }
-
```

Comment:
why is this removed here?

we still need this if the fcu.head is new?

Reply:
whoops, artifact from an earlier change, we still need this yeah